### PR TITLE
cli: html stories homogenization

### PR DESCRIPTION
**Pull request by tooppaaa**
Issue:

We need similar stories after generation to run e2e against

Part of #10702

### CODE REVIEW

**Review by ndelangen — APPROVED** [review]

File: lib/cli/src/generators/HTML/template-csf/stories/0-Welcome.stories.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  <p>
+    See these sample
+    <a (click)="showApp.emit($event)" role="button" tabIndex="0">stories</a> for a component
+    called <span class="inline-code">Button</span> .
+  </p>
+  <p>
+    Just like that, you can add your own components as stories. <br />
+    You can also edit those components and see changes right away. <br />
+    (Try editing the <span class="inline-code">Button</span> stories located at
+    <span class="inline-code">src/stories/index.js</span>.)
```
Comment (yannbf) [inline lib/cli/src/generators/HTML/template-csf/stories/0-Welcome.stories.js:25]: the path should refer to `1-Button.stories.js `

File: lib/cli/src/generators/HTML/index.js [hunk 2]
```diff
@@ -9,7 +9,13 @@ import {
 import { STORY_FORMAT } from '../../project_types';
 
 export default async (npmOptions, { storyFormat }) => {
-  const packages = ['@storybook/html'];
+  const packages = [
+    '@storybook/html',
+    '@storybook/addon-actions',
+    '@storybook/addon-links',
+    '@storybook/addons',
```
Comment (yannbf) [inline lib/cli/src/generators/HTML/index.js:16]: can we check if `@storybook/addons` is necessary?

**Review by yannbf — CHANGES_REQUESTED** [review]
LGTM! only a simple thing to change

File: lib/cli/src/generators/HTML/template-csf/stories/0-Welcome.stories.js (hunk 1 above)
Comment (gaetanmaisse) [reply lib/cli/src/generators/HTML/template-csf/stories/0-Welcome.stories.js]: Done

File: lib/cli/src/generators/HTML/index.js (hunk 2 above)
Comment (gaetanmaisse) [reply lib/cli/src/generators/HTML/index.js]: Not needed, see https://github.com/storybookjs/storybook/pull/10703#issuecomment-628227892

**Review by yannbf — APPROVED** [review]
Awesome stuff!

**Pull request merged**